src/components/custom-field-input-multiple-choice/custom-field-input-multiple-choice.jsx

```diff
@@ -227,38 +298,15 @@ const CustomFieldInputMultipleChoice = forwardRef(function CustomFieldInputMulti
             </div>
           </div>
         </div>
-        {expanded && (visibleChoices.length === 0 ? (<NoOptions className={styles['no-options']} id={ids.emptyMessage} />) : (
-          <Listbox
-            className={styles['popup-container']}
-            id={ids.listbox}
-            labelledBy={ids.label}
-            refs={choicesRefs}
-          >
-            {visibleChoices.map((choice, index) => (
-              <ListOption
-                key={`${props.id}-${choice.id}`}
-                onSelect={onChoiceSelect}
-                ref={choicesRefs[index]}
-                value={choice}
-              >
-                {choice.label}
-              </ListOption>
-            ))}
-          </Listbox>)
-        )}
+        {dropdownContents()}
       </FormControl>
     </div>
   );
 });
 
-const ChoiceType = PropTypes.shape({
-  id: PropTypes.number.isRequired,
-  label: PropTypes.string.isRequired,
-});
-
 CustomFieldInputMultipleChoice.propTypes = {
-  choices: PropTypes.arrayOf(ChoiceType).isRequired,
```

Comment:
@bennapp I think this makes sense, if we have a use case for it. If we don't, I'd prefer to keep the props API as clean as possible.

Reply:
My concern is that this will be a requirement for anyone using this within bigmaven.

Reply:
@juanca @bennapp here's the new story to capture this work when we have a use case (or otherwise prioritize it): https://www.pivotaltracker.com/story/show/177604300